json-rpc/Cargo.toml

```diff
@@ -37,6 +37,12 @@ libra-temppath = { path = "../common/temppath", version = "0.1.0", optional = tr
 storage-proto = { path = "../storage/storage-proto", version = "0.1.0" }
 
 [dev-dependencies]
+proptest = "0.9.2"
+reqwest = { version = "0.10.4", features = ["blocking", "json"], default_features = false }
+libra-crypto = { path = "../crypto/crypto", version = "0.1.0" }
```

Comment:
surprised libra-crypto and libra-temppath need referenced here as they are referenced above in the dependency section and not specifying "fuzzing"?

Reply:
`libra-temppath` is optional in dependencies, but we need it all the time here. reqwest and libra-crypto were probably merge artifacts. I've removed them. Good catch.